contrib/scripts/cilium-api-check.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Find all files with API handlers for non-GET APIs
+API_HANDLERS=( $(grep -Rl "Handle[^) ]* [^G][^.]*Param" daemon/cmd/*.go) )
```

Comment:
The path here might end up breaking in future if we refactor the API handlers (cc @joamaki). I just noticed this also doesn't catch the cilium-health API which also has a PUT, since that is under `pkg/health/server/status.go`.

Maybe we can change the filepath filter here, I'm trying to avoid `api/`, `Documentation/` and `vendor/` directories and tags in the root of the repository.

Reply:
^ Fixing in the newest version with `grep --exclude-dir` :sparkles: